tests/ui_tests/app/page/admin/page.py

```diff
@@ -570,35 +570,49 @@ def fill_select_in_policy_popup(self, items, available_items):
 
     @allure.step('Fill second step in new policy')
     def fill_second_step_in_policy_popup(
-        self, clusters: Optional[str], services: Optional[str], parent: Optional[str], providers: Optional[str]
+        self,
+        clusters: Optional[str],
+        services: Optional[str],
```

Comment:
You can pass here multiple cluster/services as single string?

Reply:
yep, supposed so

Reply:
but in code, we don't separate this string into multiple choice. I suggest renaming it to a singular form

Reply:
no, we separate them by ", "

Reply:
ah, my bad, got it